src/flick/notification/models.py

```diff
@@ -12,6 +12,7 @@ class Notification(models.Model):
         ("incoming_friend_request_accepted", "Incoming Friend Request Accepted"),
         ("outgoing_friend_request_accepted", "Outoing Friend Request Accepted"),
         ("comment_like", "Comment Like"),
+        ("lst_like", "List Like"),
```

Comment:
i see that earlier you referenced `list_like` instead of `lst_like`, could you update this one to `list_like`?